src/xrpld/app/tx/detail/applySteps.cpp

```diff
@@ -321,18 +348,31 @@ preclaim(
     std::optional<PreclaimContext const> ctx;
     if (preflightResult.rules != view.rules())
     {
-        auto secondFlight = preflight(
-            app,
-            view.rules(),
-            preflightResult.tx,
-            preflightResult.flags,
-            preflightResult.j);
+        auto secondFlight = [&]() {
```

Comment:
Is it possible to write a test to cover this code? It's a bit unclear since it looks like it wasn't covered before either.

Reply:
Looks like it's a pretty niche case, so not a big deal then.